dags/veda_data_pipeline/veda_dataset_pipeline.py

```diff
@@ -98,4 +115,5 @@ def build_stac_task(payload):
     submit_stac = submit_to_stac_ingestor_task.expand(built_stac=build_stac)
 
     collection_grp.set_upstream(start)
+    mutate_payload_task.set_upstream(start)
```

Comment:
should `collection_grp` aslo be upstream of mutate_payload_task so we don't strip assets before the collection is created?

Reply:
Nope - it's not an in-place mutation, the output gets passed to the discover step independantly.

Reply:
Resolving--the discover task, which includes the `alt_payload=mutate_payload_task()` parameter, depends on `collection_grp` already